src/functions/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Shouldn't we move this to another repository, if it is a sub-project like this?

Reply:
This is the convention when writing firebase functions. We dont need to npm install the directory.